packages/config/config/devices/0x031e/lzw45.json

```diff
@@ -0,0 +1,301 @@
+			"label": "Default Level (Z-Wave)",
+			"description": "Default level for the dimmer when it is powered on from a Z-Wave command (i.e. BasicSet(0xFF)). A setting of 0 means that the switch will return to the level that it was on before it was turned off.\n      Range: 0-99\n      Default: 0",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Default Color",
+			"description": "Byte(3-2): Values between 2700-6500 represent a color temperature. \n      Byte(1-0): Values between 0-361, represent the color on the Hue color wheel. The value of 361 represents a random color.\n      Range: 0 - 425984362\n      Default: 262144000",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 425984362,
+			"defaultValue": 262144000,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "State after power Restored",
+			"description": "The state the switch should return to once power is restored after power failure.\n      0 - Off\n      1 - Default Color / Level (Parameter 9)\n      2 - Previous\n      Default: Previous",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Default Color/Level",
+					"value": 1
+				},
+				{
+					"label": "Previous",
+					"value": 2
+				}
+			]
+		},
+		"17": {
+			"label": "Active Power Reports",
+			"description": "The power level change that will result in a new power report being sent. The value is a percentage of the previous report. 0 = disabled.\n      Range: 0 - 100\n      Default: 10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"18": {
+			"label": "Periodic Power & Energy Reports",
+			"description": "Time period between consecutive power & energy reports being sent (in seconds). The timer is reset after each report is sent.\n      Range: 0, 30 - 32767\n      Default: 3600",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 3600,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "Energy Reports",
+			"description": "The energy level change that will result in a new energy report being sent (in kWh)\n\t  0 = 0.01kWh, 10=0.1kWh, 100=1kWh\n      Range:0-127\n      Default: 10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21": {
+			"label": "Quick Strip Effect",
```

Comment:
Maybe this should be a partial param? Do you know the details in website about the entries here?

Reply:
Depends on the entries, but this certainly looks like it